### PR TITLE
Feature/adding placeholder prop

### PR DESCRIPTION
**Pull request by ColinSpencelyWork**
Added an optional placeholder prop to searchbar. If prop is not provided, the default placeholder is 'Search...'.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "name": "@pderas/vue2-searchbar",
+  "version": "1.1.2",
```
Comment (Mar4elius) [inline package-lock.json]: I'm just curious, wasn't it here before? Shouldn't you just update `version`?

**Review by Mar4elius — APPROVED** [review]

**Comment by reed-jones** [comment]
you gotta delete the node_module files

File: package-lock.json (hunk 1 above)
Comment (reed-jones) [reply package-lock.json]: when you update in package.json, it should automatically update here too (I'm assuming you didn't write this colin?

Comment (ColinSpencelyWork) [reply package-lock.json]: I did not write this. First time messing with such things...

**Review by reed-jones — APPROVED** [review]

**Pull request merged**